components/Wallet/Send.js/GasCustomization.jsx

```diff
@@ -8,7 +8,8 @@ import {
   Label,
```

Comment:
Line 109 I get this error:

`Unhandled Rejection (TypeError): Cannot read property 'estimateGas' of null`

```
106 | 
  107 | const [attemptingTx, setAttemptingTx] = useState(false)
  108 | 
> 109 | const estimateGas = async gp => {
      | ^  110 |   // create a fake message
  111 |   const message = new Message({
  112 |     to: wallet.address,
```

Reply:
`npm i @openworklabs/filecoin-wallet-provider@latest` try that, i updated the filecoin-wallet-provider